website/content/docs/operations/metrics.mdx

```diff
@@ -389,7 +389,7 @@ those listed in [Key Metrics](#key-metrics) above.
 | `nomad.nomad.plan.evaluate`                          | Time elapsed to evaluate a plan                                   | Nanoseconds          | Summary | host                         |
 | `nomad.nomad.plan.queue_depth`                       | Count of evals in the plan queue                                  | Integer              | Gauge   | host                         |
 | `nomad.nomad.plan.submit`                            | Time elapsed for `Plan.Submit` RPC call                           | Nanoseconds          | Summary | host                         |
-| `nomad.nomad.plan.wait_for_index`                    | Time elapsed for the planner to obtain a snapshot                 | Nanoseconds          | Summary | host                         |
+| `nomad.nomad.plan.wait_for_index`                    | Time elapsed the planner waits for plan's index                   | Nanoseconds          | Summary | host                         |
```

Comment:
Given...
> The most important thing about the metric is what we're waiting on
> before taking a snapshot: the raft index of the object to be
> processed (plan or eval).

and that this text will wrap nicely when rendered, I'd be a bit more verbose here and say something like:

> Time elapsed that planner waits for the raft index of the plan to be processed


(And similar for the one below)

Reply:
For the absolutely silliest reason:

![image](https://user-images.githubusercontent.com/113362/121064809-e2361780-c77c-11eb-819d-19e1edcff495.png)

Should I go ahead and resize the whole table or just let these two rows be different?

Reply:
Oh, if I hit `<tab>` in emacs markdown mode it just magically aligns the whole table for me. 😀  But yeah that means it'll wrap wildly in an editor. That's unfortunate about markdown tables. Probably not worth worrying about.

Reply:
I pushed the wording change. The columns are realigned but the whole row isn't bumped out.